packages/nestjs/package.json

```diff
@@ -26,35 +28,48 @@
         "types": "./build/types/index.d.ts",
         "default": "./build/cjs/index.js"
       }
-    }
-  },
-  "typesVersions": {
-    "<4.9": {
-      "build/types/index.d.ts": [
-        "build/types-ts3.8/index.d.ts"
-      ]
+    },
+    "./setup": {
+      "import": {
+        "types": "./setup.d.ts",
+        "default": "./build/esm/setup.js"
+      },
+      "require": {
+        "types": "./setup.d.ts",
+        "default": "./build/cjs/setup.js"
+      }
     }
   },
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
     "@sentry/core": "8.19.0",
-    "@sentry/node": "8.19.0"
+    "@sentry/node": "8.19.0",
+    "@sentry/types": "8.19.0",
+    "@sentry/utils": "8.19.0"
+  },
+  "devDependencies": {
+    "@nestjs/core": "^10.3.10",
+    "@nestjs/common": "^10.3.10"
+  },
+  "peerDependencies": {
+    "@nestjs/core": "^10.3.10",
+    "@nestjs/common": "^10.3.10"
```

Comment:
h: Just compatibility wise it would be quite important to find the lowest possible NestJS version compatible with this PR. The way we have it right now, people would have to be on the very latest NestJS version which we can or should not expect from them.

Reply:
Updated: Seems to be compatible with nest version 8 upwards